### PR TITLE
clean Makefile to simplify env

### PR DESCRIPTION
**Pull request by qingyunha**
Signed-off-by: Tao Qingyun <845767657@qq.com>

**1.Describe what this PR did**
use `go env` for `GOPATH` and little cleanup in Makefile
**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # Go parameters
-GOCMD=go
-GOBUILD=$(GOCMD) build
-GOCLEAN=$(GOCMD) clean
-GOTEST=$(GOCMD) test
-GOPATH=$(shell cd ../../../..; pwd)
-GOPACKAGES=$(shell GOPATH=`cd ../../../..; pwd` go list ./... | grep -v /vendor/ | sed 's/^_//')
```
Comment (allencloud) [inline Makefile]: I agree with all your change above except this one `| sed 's/^_//'`.
Could you give me a reason that why you eliminate this part?
Thanks a lot. @qingyunha

Comment (qingyunha) [reply Makefile]: Maybe I am wrong. I don't see any package name starts with '_'.

Comment (allencloud) [reply Makefile]: So could you keep this part as it was.

Comment (qingyunha) [reply Makefile]: Ok. updated

**Comment by allencloud** [comment]
LGTM, thanks.

**Pull request merged**